lib/job.js

```diff
@@ -345,6 +345,17 @@ class Job extends BaseJob {
     return this.client.call('timeout', this.jid);
   }
 
+  /**
+   * If the job is in the `scheduled` state, return a promise that resolves to when
+   * it's scheduled to run. Otherwise, return a promise that resolves to null.
```

Comment:
Perhaps because I'm new to this codebase, but it'd be beneficial to know that the scheduled time is the number of milliseconds since epoch.

Reply:
Done :-)

Reply:
thanks!